index.html

```diff
@@ -296,6 +296,8 @@
           resize(key);
         } else if (key === "1" || key === "2" || key === "3" || key === "4") {
           subdivide_grid(key);
+        } else if (key === "Escape") {
+          activate_tool(active_tool);
```

Comment:
This works, but it looks like a hack, you don't want to reactivate the tool, what if this function did something fancy, like animation in the toolbox?

The appropriate function call would be `clear_temporary_elements()`, which is the first thing that `activate_tool` does.